ap_api/callbacks/customer/index.mdx

```diff
@@ -5,16 +5,19 @@ order: 0
 
 import {EndpointsTable} from "@site/src/components/EndpointsTable";
 
+Customers callback are designed to work with a customer data. When working with the Anchor Platform, business will store KYC data defined by [SEP-12].
+
+However, because the Anchor Platform does not store this data, it's required to fetch it from the business server. This set of endpoints allows to forward data stored on the business side to the wallet, in the SEP format.
```

Comment:
English tweaks, write as: "However, because the Anchor Platform does not store this data, it must fetch it from the business server. This set of endpoints allows the Anchor Platform to forward data stored on the business side to the wallet in SEP format."